embassy-stm32/build.rs

```diff
@@ -81,6 +81,18 @@ fn main() {
         singletons.push(c.name.to_string());
     }
 
+    // ========
+    // Handle low power.
+
+    let low_power = match env::vars()
```

Comment:
it's probably easier to unconditionally codegen code that has `#[cfg(feature="low_power")]`